### PR TITLE
storaged: Migrate this code to build with webpack

### PR DESCRIPTION
**Pull request by stefwalter**
Blocking on #4687

### CODE REVIEW

File: pkg/storaged/client.js [hunk 1]
```diff
@@ -17,19 +17,28 @@
...
+
+    var $ = require('jquery');
+    var cockpit = require('cockpit');
+
+    var utils = require('./utils');
 
     /* STORAGED CLIENT
      */
 
+    /* HACK: https://github.com/storaged-project/storaged/pull/68 */
```
Comment (larsu) [inline pkg/storaged/client.js]: This seems like an unrelated change.

Comment (stefwalter) [inline pkg/storaged/client.js]: It's related. The whole way the manifests are used here is a HACK around storaged. I just added the comment to clarify what's going on. Should I put the comment in a separate pull request. In any case I'll rebase this onto #4915 and that should help this work better.

Comment (stefwalter) [inline pkg/storaged/client.js]: Simplified this code by rebasing on #4915. But the comment remains.
